### PR TITLE
fix(react-15): exclude prop types from html attributes

### PR DESCRIPTION
**Pull request by nLight**
This PR addresses all the react 15.4.x warnings by filtering the attributes we provide to tags.

### CODE REVIEW

**Comment by nLight** [comment]
🙇  It is an anti-pattern, one step at a time. Gotta test it with DC/OS UI thus WIP.

**Comment by nLight** [comment]
Tested, it should be good to review

File: src/Form/FieldCheckbox.js [hunk 1]
```diff
@@ -87,13 +87,19 @@ class FieldCheckbox extends FieldRadioButton {
       attributes.indicatorClass
     );
 
+    const htmlAttributes = Util.pick(attributes, [
```
Comment (ahoskins) [inline src/Form/FieldCheckbox.js]: Should use `exclude()` here to be consistent with the others.

File: src/List/ListItem.js [hunk 2]
```diff
@@ -8,11 +8,12 @@ class ListItem extends React.Component {
     const { props } = this;
     const Tag = props.tag;
 
+    const excludeProps = ["children", "className", "tag", "transition"];
```
Comment (ahoskins) [inline src/List/ListItem.js]: In others the excluded fields are inline, should also be inline here!

File: src/Form/FieldInput.js [hunk 3]
```diff
@@ -214,6 +219,7 @@ FieldInput.defaultProps = {
   columnWidth: 12,
   handleEvent() {},
   value: "",
+  startValue: "",
```
Comment (ahoskins) [inline src/Form/FieldInput.js]: This seems to have snuck in, and not actually in scope of PR?

Comment (nLight) [inline src/Form/FieldInput.js]: BUSTED! 👮  
I think that is a residue from an attempt to fix switching uncontrolled to a controlled input. Maybe it even partially works, I'll double check.

File: src/Form/FieldCheckbox.js (hunk 1 above)
Comment (nLight) [inline src/Form/FieldCheckbox.js]: I'd prefer having `pick` over `exclude`. Allowing what is for sure ok to allow is more strict and less error prone.

**Comment by nLight** [comment]
Hey @ahoskins I addressed your comments! 🙇

**Pull request merged**